gavel/controllers/judge.py

```diff
@@ -86,6 +86,13 @@ def tx():
             elif request.form['action'] == 'Absent':
                 annotator.next.absent.append(annotator)
                 annotator.ignore.append(annotator.next)
+                if count >= settings.THRESHOLD_ABSENT:      #from here
```

Comment:
Did you test this out? `count` doesn't seem to be defined anywhere, and it should give you an error the moment you run this. You should be looking at the length of the absent list: `len(annotator.next.absent)`.